src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs

```diff
@@ -77,5 +77,24 @@ public override bool TryGetDisplayName(DName logicalName, out DName displayName)
         {
             return _logicalToDisplay.TryGetValue(logicalName, out displayName);
         }
+
+        public DisplayNameProvider RemoveField(DName lookupName)
```

Comment:
>RemoveField [](http://example.com/codeflow?start=35&length=11)

Do we get full code-coverage on this method?

Reply:
I have added test for the same.